src/Orleans.Runtime/Streams/PersistentStream/PersistentStreamPullingAgent.cs

```diff
@@ -371,10 +376,10 @@ public void RemoveSubscriber_Impl(GuidId subscriptionId, StreamId streamId)
             catch (Exception exc)
             {
                 receiverInitTask = null;
-                logger.Error(ErrorCode.PersistentStreamPullingAgent_12, "Exception while PersistentStreamPullingAgentGrain.AsyncTimerCallback", exc);
+                logger.Error(ErrorCode.PersistentStreamPullingAgent_12, $"Exception while trying to read from queue {queueId}", exc);
```

Comment:
Think we should be clear that this failure is after retries.  Maybe "Giving up reading from queue {queueId} after retry attempts." or something of the sort.